content/en/docs/tasks/administer-cluster/highly-available-control-plane.md

```diff
@@ -133,7 +133,7 @@ the [etcd administration guide](https://etcd.io/docs/v2.3/admin_guide/#member-mi
 
 ## Implementation notes
 
-![ha-master-gce](/images/docs/ha-master-gce.png)
+![ha-control-plane](/images/docs/ha-control-plane.svg)
 
 ### Overview
 
```

Comment:
The figure above illustrates three control plane nodes and their components in a highly available cluster. The control plane node’s components employ the following methods:

- etcd: instances are clustered together using consensus.

- Controllers, scheduler and cluster auto-scaler: only one instance of each will be active in a cluster using a lease mechanism.

- Add-on manager: each works independently to keep add-ons in sync.

In addition, a load balancer operating in front of the API servers routes external and internal traffic to the control plane nodes.